src/sentry/integrations/github_enterprise/webhook.py

```diff
@@ -0,0 +1,439 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import dateutil.parser
+import hashlib
+import hmac
+import logging
+import six
+
+from django.db import IntegrityError, transaction
+from django.http import HttpResponse
+from django.utils.crypto import constant_time_compare
+from django.utils.decorators import method_decorator
+from django.views.decorators.csrf import csrf_exempt
+from django.views.generic import View
+from django.utils import timezone
+from simplejson import JSONDecodeError
+from sentry.models import (
+    Commit, CommitAuthor, CommitFileChange, Identity, Integration, PullRequest,
+    Repository, User
+)
+from sentry.utils import json
+from sentry.constants import ObjectStatus
+from sentry.integrations.exceptions import ApiError
+from .repository import GitHubEnterpriseRepositoryProvider
+from .client import GitHubEnterpriseAppsClient
+
+logger = logging.getLogger('sentry.webhooks')
+
+
+def is_anonymous_email(email):
+    return email[-25:] == '@users.noreply.github.com'
```

Comment:
i wonder if this will actually be `github.com` for the enterprise version too?

Reply:
Yeah Im not sure what we want this to be